app/views/posts/_allposts.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render partial: 'users/user', locals: { user: @user } %>
+
+<section id="posts-container">
+    <% @user.posts.each do |post| %>
+      <%= render partial: 'posts/post', locals: { post: post } %>
+      <div style="border: 2px solid grey;">
+      <% if post.comments.present? %>
+      
+          <% post.recent_comments do |comment| %>
+            <%= render partial: 'comments/commets', locals: { comment: comment } %>
+          <% end %>
+          <% else %>
+            <p>No comments yet.</p>
+          <% end %>
```

Comment:
👍 Great job on incorporating the feedback! There's a further suggestion to enhance the functionality:

- [ ] **Display Recent Comments:** In your recent update, the `recent_comments` method is now being used. However, it seems that there might be an issue with the display of comments in your view. Make sure that the view is configured to show the recent comments for each post. If there are comments associated with a post, ensure they are visible to the user in the expected format.
![image](https://github.com/bhushan354/Blog-App-Rails/assets/55840999/75ffc35c-ea41-45d4-9528-c984e860cd10)


By addressing this, you'll provide users with a better experience by allowing them to see recent comments associated with each post. Keep up the good work! 🚀👏